dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -1219,4 +1139,85 @@ protected DoraOpenFileHandleContainer getOpenFileHandleContainer() {
   public WorkerNetAddress getAddress() {
     return mAddress;
   }
+
+  /**
+   * Build FileInfo from UfsStatus and UFS full Path.
+   *
+   * @param cacheUsage
+   * @param ufsType
+   * @param status
+   * @param ufsFullPath
+   * @param xattrMap
+   * @return a FileInfo
+   */
+  public static alluxio.grpc.FileInfo buildFileInfoFromUfsStatus(
```

Comment:
there's probably a better place for these 2 static methods but I don't see it right now